lib/rules/no-old-shims.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const oldShimsData = require('ember-rfc176-data/old-shims.json');
+
+//------------------------------------------------------------------------------
+// General rule - Don't use import paths from ember-cli-shims
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {},
+    fixable: 'code',
+  },
+
+  create(context) {
+    const message = 'Don\'t use import paths from ember-cli-shims';
+
+    return {
+      ImportDeclaration(node) {
+        const source = node.source.value;
```

Comment:
What do you think about `packageName` or `moduleName` instead of `source`? For a person that looks at this code for the first time it will probably be more understandable. It's just a suggestion though